terraform/environments/digital-prison-reporting/application_variables.json

```diff
@@ -14,9 +14,15 @@
       "reporting_hub_batch_duration_seconds": 30,
       "reporting_hub_add_idle_time_between_reads": true,
       "reporting_hub_idle_time_between_reads_in_millis": 140,
+      "reporting_hub_retry_max_attempts": 10,
+      "reporting_hub_retry_min_wait_millis": 100,
+      "reporting_hub_retry_max_wait_millis": 10000,
       "refresh_job_worker_type": "G.1X",
       "refresh_job_num_workers": 2,
       "refresh_job_log_level": "INFO",
+      "maintenance_job_retry_max_attempts": 10,
+      "maintenance_job_retry_min_wait_millis": 1000,
+      "maintenance_job_retry_max_wait_millis": 20000,
```

Comment:
The time spent waiting on each iteration with these setting is roughly like this:

```
Retrying after attempt 1. Total time spent waiting: 1,121ms.
Retrying after attempt 2. Total time spent waiting: 3,296ms.
Retrying after attempt 3. Total time spent waiting: 6,489ms.
Retrying after attempt 4. Total time spent waiting: 14,194ms.
Retrying after attempt 5. Total time spent waiting: 32,057ms.
Retrying after attempt 6. Total time spent waiting: 57,081ms.
Retrying after attempt 7. Total time spent waiting: 75,936ms.
Retrying after attempt 8. Total time spent waiting: 92,927ms.
Retrying after attempt 9. Total time spent waiting: 111,546ms.
Retries exceeded on attempt 10. Total time spent waiting: 111,548ms.
```